guides/bug_report_templates/active_record_migrations_master.rb

```diff
@@ -0,0 +1,64 @@
+begin
+  require "bundler/inline"
+rescue LoadError => e
+  $stderr.puts "Bundler version 1.10 or later is required. Please update your Bundler"
+  raise e
+end
+
+gemfile(true) do
+  # source "https://rubygems.org"
+  gem "activerecord"
```

Comment:
We should use `gem "rails", github: "rails/rails"` here similar to how it is used in other master bug report templates.